### PR TITLE
Min1

### PR DESCRIPTION
**Pull request by min2028**
Google oauth

### CODE REVIEW

**Comment by min2028** [comment]
Weird my latest commits are not showing up

**Review by kmdicksn — APPROVED** [review]
lgtm

File: frontend/src/components/landingPage/AuthButton.js [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+            dispatch(getUserAsync(credential)).then(() => {
+                console.log('User after dispatch:', user);
+            });
+        }
+    };
+    useEffect(() => {
+        console.log('User in useEffect:', user);
+    }, [user]);
+
+    console.log('User before rendering:', user);
```
Comment (kmdicksn) [inline frontend/src/components/landingPage/AuthButton.js]: can we remove the console logs?

Comment (min2028) [reply frontend/src/components/landingPage/AuthButton.js]: yes

**Review by kmdicksn — APPROVED** [review]

**Pull request merged**